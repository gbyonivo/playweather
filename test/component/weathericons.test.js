import React from 'react';
import renderer from 'react-test-renderer';
import WeatherIcons from '../../src/components/weatherIcons';

describe('Weather component without test', () => {
  it('renders correctly when isWithText is false', () => {
    const actual = renderer.create(
      <WeatherIcons weather={[{ icon: '10d', main: 'cloudy' }]}/>
    );
    expect(actual).toMatchSnapshot();
  });
});

describe('WeatherIcons component withText', () => {
  it('renders correctly when isWithText is true', () => {
    const actual = renderer.create(
      <WeatherIcons weather={[{ icon: '10d', main: 'cloudy' }]} isWithText/>
    );
    expect(actual).toMatchSnapshot();
  });
});
