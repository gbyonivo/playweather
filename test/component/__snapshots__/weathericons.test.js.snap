// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather component without test renders correctly when isWithText is false 1`] = `
<div
  className="weatherIcons"
>
  <div
    className="weatherIcon"
  >
    <img
      alt=""
      src="http://openweathermap.org/img/w/10d.png"
    />
  </div>
</div>
`;

exports[`WeatherIcons component withText renders correctly when isWithText is true 1`] = `
<div
  className="weatherIcons"
>
  <div
    className="weatherIcon"
  >
    <div
      className="text"
    >
      cloudy
    </div>
    <img
      alt=""
      src="http://openweathermap.org/img/w/10d.png"
    />
  </div>
</div>
`;
